### ISSUE TITLE
Tests fail with in regard to datetime values

### ISSUE THREAD

**Issue by smoore-p4** [Paradigm4/ArrayOpR#54]
```
── Failure ('test_array_op_from_df.R:259:3'): upload data frame with other scidbR settings ──
`dbdf1` not equal to `df`.
Component "f_datetime": Mean absolute difference: 0.04887795
── Failure ('test_array_op_from_df.R:260:3'): upload data frame with other scidbR settings ──
`dbdf2` not equal to `df`.
```

Could be due to different times between the scidb docker image and the host machine?

**Comment by jwimberl** [comment]
This is addressed by #56 (containing within in #51)

**Comment by jwimberl** [comment]
This is addressed by #56 (containing within in #51)
